### DISCUSSION TITLE
Webhook icon

### DISCUSSION THREAD

**Discussion by rigtigeEmil** [tailwindlabs/heroicons#1114] [Ideas]
Most modern applications allow you to setup webhooks, to receive callbacks when some event happens in a given system. 
Might look something like this:
![image](https://github.com/tailwindlabs/heroicons/assets/70773323/01cd6d12-bf49-419a-98e7-eec088638250)

**Comment by micah-redwood** [comment]
Great idea! A similar use case I've run into is trying to indicate that the "user" behind an action was an automated system, not a person. Might be helpful to have a robot icon or some other icon indicating an automated process.
